src/Components/ErrorToast/ErrorToast.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ToastContainer, toast } from "react-toastify";
+
+import "react-toastify/dist/ReactToastify.css";
+
+export const showError: (msg: string) => React.ReactText = (msg: string) =>
```

Comment:
I'd prefer not to export functions and components from the same file. In your structure, this should probably live in `utils`